diff.py

```diff
@@ -697,12 +700,19 @@ def apply_format(self, chunk: str, f: Format) -> str:
             ]
         else:
             static_assert_unreachable(f)
-        return f"{ansi_code}{chunk}{Style.RESET_ALL}"
+        return f"{ansi_code}{chunk}{Fore.RESET}"
 
     def table(self, meta: TableMetadata, lines: List[Tuple["OutputLine", ...]]) -> str:
-        rows = [meta.headers] + [self.outputline_texts(ls) for ls in lines]
+        rows = [(meta.headers, False)] + [
+            (self.outputline_texts(line), line[1].is_data_ref) for line in lines
+        ]
         return "\n".join(
-            "".join(self.apply(x.ljust(self.column_width)) for x in row) for row in rows
+            "".join(
+                (Back.BLACK if is_data_ref else "")
```

Comment:
Could do invert instead of black background, so it's more likely to be readable? (`\u001b[7m` to enable; `\u001b[27m` to disable)

Reply:
It looks a bit more jarring than the subtle black background:
![image](https://user-images.githubusercontent.com/371089/132401850-d54e9752-b222-4435-a94d-e87723f3f43d.png)

but it may be fine. Let's try it, I agree it should be more portable.